charts/castai-sec-agent/values.yaml

```diff
@@ -0,0 +1,102 @@
+# Default values for castai-sec-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 2
```

Comment:
I think we talked in short chat in the morning, replica 1 + leader election disabled